crates/astria-conductor/src/executor/builder.rs

```diff
@@ -52,14 +52,14 @@ impl ExecutorBuilder<WithRollupAddress, WithShutdown> {
         let WithRollupAddress(rollup_address) = rollup_address;
         let WithShutdown(shutdown) = shutdown;
 
-        let (firm_blocks_tx, firm_blocks_rx) = mpsc::channel(16);
-        let (soft_blocks_tx, soft_blocks_rx) = mpsc::channel(16);
+        let (firm_block_tx, firm_block_rx) = mpsc::channel(16);
```

Comment:
Leaving this at 16 for now. This is just a simple backpressure mechanism that ideally should not even be triggered because executor pulls firm blocks with a higher priority than soft blocks.